src/Entity/Format.php

```diff
@@ -8,7 +8,7 @@ class Format extends AbstractEntity
 {
     public function getFormat(): ?string
     {
-        return $this->get('format');
+        return $this->get('format_id');
```

Comment:
It looks like that you have changed this in a wrong method :)

Reply:
You're right. It looks like it was an extra bottle of beer yesterday...